app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettingsFragment.kt

```diff
@@ -38,7 +38,7 @@ class WallpaperSettingsFragment : Fragment() {
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? {
```

Comment:
Generally speaking, we try to have our patches here be as small and focused as possible. Normally a small, unrelated change like this would be something we try to avoid. This is still a useful change, so I am okay with keeping it here but wanted to call it out for future reference.

Reply:
Yes i understand. Will keep in mind for future reference. Thanks. 
For now i have reverted it. At that time i wasn't able to build due to this warning.